### ISSUE TITLE
Dump `iggy-bench` results to csv

### ISSUE THREAD

**Issue by hubcio** [apache/iggy#229]
Append them to given csv, and if possible generate excel file with relevant charts.

**Comment by Molter23** [comment]
I would like to work on it.

**Comment by BartoszCiesla** [comment]
Go ahead @Molter23 , it's assigned to you now

**Comment by BartoszCiesla** [comment]
Yes, that's good idea. Basic use case would be to call bench multiple times so file would have to be updated with each run in order to have data from all the tests. Then charts could be created by importing to spreadsheet or using tools like GNUplot or other tools.
